superset-frontend/src/SqlLab/components/SqlEditor/index.jsx

```diff
@@ -253,6 +262,8 @@ const SqlEditor = ({
   const sqlEditorRef = useRef(null);
   const northPaneRef = useRef(null);
 
+  const SqlAIExtension = extensionsRegistry.get('sqleditor.ai.form');
```

Comment:
After working on the database connections modal extension, I would recommend calling the extensionRegistry in the top most component that you think will need to have any access to state or any controls that you may want to pass in or retrieve later.

Reply:
Here's an example: https://github.com/apache/superset/pull/23174/files#diff-839cf9575c4ae323ef0e684535a3edc6b549e845627b31066fedf81607e795b7R589